Maskbook/Wallet/WalletService/PendTransacation/PendTransactionManager.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  PendTransactionManager.swift
+//  Maskbook
+//
+//  Created by caiyu on 2022/4/7.
+//  Copyright © 2022 dimension. All rights reserved.
+//
+
+import Foundation
+import web3swift
+import Combine
+import BigInt
+
+class PendTransactionManager {
+
+    private var pendList:[PendTransactionModel] = []
+    private var timer: Timer?
+    var subscriptions: Set<AnyCancellable> = []
+    public static let shared = PendTransactionManager()
+    
+    public var pendTransactions: CurrentValueSubject<[PendTransactionModel], Never> = CurrentValueSubject([])
```

Comment:
1. `pendList` and `pendTransactions` could be 1 `CurrentValueSubject` as the unified data source IMO.
2. `subscriptions` not used?